demisto_sdk/dev_tools/lint_manager.py

```diff
@@ -0,0 +1,139 @@
+        self.configuration = configuration
+
+    def run_dev_packages(self) -> int:
+        # if we dont run in parallel we run a single process
+        if not self.parallel:
+            linter = Linter(self.pkgs[0], no_test=not self.run_args['tests'],
+                            no_pylint=not self.run_args['pylint'], no_flake8=not self.run_args['flake8'],
+                            no_mypy=not self.run_args['mypy'], verbose=self.log_verbose, root=self.root,
+                            keep_container=self.keep_container, cpu_num=self.cpu_num, configuration=self.configuration)
+
+            return linter.run_dev_packages()
+
+        # we run parallel processes
+        else:
+            max_workers = int(os.getenv("MAX_WORKERS", "10"))
+            pkg_dirs = self.pkgs
+            pkgs_to_run = []
+            for dir in pkg_dirs:
+                if self._should_run_pkg(pkg_dir=dir):
+                    pkgs_to_run.append(dir)
+
+            print("Starting parallel run for [{}] packages with [{}] max workers".format(len(pkgs_to_run), max_workers))
+            fail_pkgs = []
+            good_pkgs = []
+
+            # run CommonServer non parallel to avoid conflicts
+            # when we modify the file for mypy includes
+            if 'Scripts/CommonServerPython' in pkgs_to_run:
+                pkgs_to_run.remove('Scripts/CommonServerPython')
+                res, _ = self._run_single_package_thread(package_dir='Scripts/CommonServerPython')
+                if res == 0:
+                    good_pkgs.append('Scripts/CommonServerPython')
+
+                else:
+                    fail_pkgs.append('Scripts/CommonServerPython')
+
+            with concurrent.futures.ThreadPoolExecutor(max_workers=max_workers) as executor:
+                futures_submit = [executor.submit(self._run_single_package_thread, dir) for dir in pkgs_to_run]
+                for future in list(concurrent.futures.as_completed(futures_submit)):
+                    result = future.result()
+                    status_code = result[0]
+                    package_ran = result[1]
+                    if status_code == 0:
+                        good_pkgs.append(package_ran)
+
+                    else:
+                        fail_pkgs.append(package_ran)
+
+                if fail_pkgs:
+                    print_color("\n******* FAIL PKGS: *******", LOG_COLORS.RED)
+                    print_color("\n\t{}\n".format("\n\t".join(fail_pkgs)), LOG_COLORS.RED)
+
+                if good_pkgs:
+                    print_color("\n******* SUCCESS PKGS: *******", LOG_COLORS.GREEN)
+                    print_color("\n\t{}\n".format("\n\t".join(good_pkgs)), LOG_COLORS.GREEN)
+
+                if not good_pkgs and not fail_pkgs:
+                    print_color("\n******* No changed packages found *******\n", LOG_COLORS.YELLOW)
+
+                if fail_pkgs:
+                    sys.exit(1)
+
+    def _should_run_pkg(self, pkg_dir: str) -> bool:
```

Comment:
not a good name, not sure what it does, add docstring

Reply:
done